lib/rubocop/cop/style/unneeded_condition.rb

```diff
@@ -96,8 +96,9 @@ def else_source(else_branch)
         end
 
         def make_ternary_form(node)
-          ternary_form = [node.if_branch.source,
-                          else_source(node.else_branch)].join(' || ')
+          _condition, if_branch, else_branch = *node
```

Comment:
Looking at this change makes me think something is awry with the node extension for `IfNode` ... 🤔